.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+# Set up locally by running
+#
+# $ pre-commit install
```

Comment:
That would fit nicely in the `README.md` ;)

Reply:
Check out 49f83fb :)